ref: apple release health opt-out (#2414)
getsentry/sentry-docs: src/platforms/common/configuration/releases.mdx

Monitor the [health of releases](/workflow/releases/health/) by observing user adoption, usage of the application, percentage of [crashes](/workflow/releases/health/#crash), and [session data](/workflow/releases/health/#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.

> To benefit from the health data you must use at least version 5.0.0 of the Cocoa SDK. Since version 6.0.0, this feature is enabled by default.

Reviewer: ```suggestion
To benefit from the health data you must use at least version 5.0.0 of the Cocoa SDK. As of version 6.0.0, this feature is enabled by default.
```

The SDK will automatically manage the start and end of the sessions when the application is started, goes to background, returns to the foreground, etc.

By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:

<PlatformContent includePath="configuration/session-tracking-interval" />

> If you'd like to opt-out of this feature, you can do it via options.

Reviewer: ```suggestion
If you'd like to opt-out of this feature, you can so using options.
```
